src/it/scala/org/mdedetrich/stripe/v1/AccountIT.scala

```diff
@@ -17,11 +17,13 @@ class AccountIT extends IntegrationTest with ParallelTestExecution {
       val dob       = LocalDate.now().minusYears(30)
       val firstName = "Gaspard"
       val lastName  = "Augé"
-      val legalEntity = Some(
-        LegalEntity.default
-          .copy(`type` = Some(Individual), firstName = Some(firstName), lastName = Some(lastName), dob = Some(dob)))
+      val legalEntity = Option(
+        LegalEntity(`type` = Option(Individual),
+                    firstName = Option(firstName),
```

Comment:
`Option` is always safe, `Some` is not. I.e. if you do `Option(null)` you will get a `None`, where as with `Some(null)` you get an optional value with null inside.

I only use `Some` in situations where performance really matters